### ISSUE TITLE
limited mode: Consider adding a notification in the help menu explaining that the user is in limited mode

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#17637]
Perhaps in the help menu, we could have a little warning about "You're currently in limited acces mode. To access all of Cockpit's features, switch to administrator mode." 

It might be too wordy though.

But the general thought process is this: If someone is wondering why some feature isn't in Cockpit and they consult the help menu, then that's another place where we can warn them that they're in limited access mode and provide a hint to become an administrator.

**Comment by KKoukiou** [comment]
I suggest we address this only if we notice something from a usability review or you have a clear suggestion that we need this. Let's not keep issues open with not clear idea if it's needed or not as it's very distracting.

**Comment by martinpitt** [comment]
Agreed -- we already have this in two places (panel and Overview page). I'd rather wait until anyone actually complains about this, and then we should preferably add a tooltip or so to these two places, instead of adding yet another one.

So I am closing this for now. @garrett , please reopen if you disagree. Thanks!
